types/steamapi/index.d.ts

```diff
@@ -0,0 +1,195 @@
+// Type definitions for steamapi 2.1
+// Project: https://github.com/xDimGG/node-steamapi
+// Definitions by: vanitasboi <https://github.com/vanitasboi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class SteamAPI {
+        constructor(key: string, options?: ConstructorOptions);
+        get(path: string, base?: string, key?: string): object;
+        resolve(info: string): Promise<string>;
+        getAppList(): Promise<App[]>;
+        getFeaturedCategories(): Promise<object[]>;
+        getFeaturedGames(): Promise<object>;
+        getGameAchievements(app: string): Promise<object>;
+        getGameDetails(app: string, force?: boolean): Promise<object>;
```

Comment:
does it require optional `region` param?
https://github.com/xDimGG/node-steamapi/blob/9b973aaf354daaaaef17dcb35a2ea641460379d5/src/SteamAPI.js#L148

Reply:
yes, you are right. that would be an optional string then

Reply:
```suggestion
        getGameDetails(app: string, force?: boolean, region?: string): Promise<object>;
```